tests/test_tools.py

```diff
@@ -250,3 +251,35 @@ def get_time() -> str:
             "destructiveHint": None,
             "openWorldHint": None,
         }
+
+
+async def test_convert_langchain_tool_to_fastmcp_tool():
```

Comment:
Could you check if this works with subclassing from BaseModel? (i.e., any other ways of defining tools?)

Also what happens if a tool has injectable parameters?

Reply:
good call on both

re: injectable params - they will be dropped, since we're using `tool_call_schema`, as opposed to `args_schema`

Reply:
Any chance you could add one test case with an injectable param?